src/components/UI/Header.tsx

```diff
@@ -70,13 +70,14 @@ export default function Header({
             </span>
           </span>
         )}
-        {numberOfGames ? (
+        {numberOfGames !== undefined && numberOfGames > 0 && (
           <span className="totalGamesText">
             {t('Total Games')}: {numberOfGames}
           </span>
-        ) :
+        )}
+        {numberOfGames !== undefined && numberOfGames === 0 && (
```

Comment:
I think is best than using the location argument. 👍🏽

Reply:
Yeah this is also possible and I also think it's better. Changing it.

Reply:
I mean, keep this solution. I think useLocation approach is not good.

Reply:
I tried it but when adding a `useLocation` and using `location.pathname === "/"` instead of `numberOfGames !== undefined` it kept showing the `numberOfGames` (`0`)
![a](https://user-images.githubusercontent.com/24937357/108542646-4b843e80-72e4-11eb-88db-31a21f39023e.png)
and also TypeScript complained that `numberOfGames is possibly 'undefined'`

Reply:
Yeah. I tested here and your solution is better. Let me know when it is ready to merge.

Reply:
The 0 showing probably had something to do with the way I did the inline conditional rending.
This is the first React code base I'm working with and I honestly don't like jsx/tsx. It feels weird coming from Vue where it works [a little different](https://v3.vuejs.org/guide/conditional.html#v-if)